packages/frontend/amp/components/Standfirst.tsx

```diff
@@ -22,47 +23,65 @@ const listStyles = (pillar: Pillar) => css`
         height: 12px;
         width: 12px;
         margin-right: 8px;
-        background-color: ${palette.neutral[86]};
+        background-color: ${pillarPalette[pillar].neutral.border};
         margin-left: -20px;
     }
 `;
 
-const standfirstCss = pillarMap(
-    pillar => css`
-        ${headline(2)};
-        font-weight: 100;
-        color: ${palette.neutral[7]};
-        margin-bottom: 12px;
-        ${listStyles(pillar)};
-        p {
-            margin-bottom: 8px;
-            font-weight: 200;
-        }
-        strong {
-            font-weight: 700;
-        }
-    `,
-);
+const standfirstCss = (pillar: Pillar) => css`
+p,
+    ${headline(2)};
+    font-weight: 100;
+    color: ${palette.neutral[7]};
+    margin-bottom: 12px;
+    margin-bottom: 12px;
+    ${listStyle(pillar)};
+    p {
+        margin-bottom: 8px;
+        font-weight: 200;
+    }
+    strong {
+        font-weight: 700;
+    }
+`;
 
 const standfirstLinks = pillarMap(
     pillar =>
         css`
             a {
                 color: ${pillarPalette[pillar].dark};
                 text-decoration: none;
-                border-bottom: 1px solid ${palette.neutral[86]};
+                border-bottom: 1px solid ${pillarPalette[pillar].neutral.border};
+            }
+            a:hover {
+                border-bottom: 1px solid ${pillarPalette[pillar].dark};
             }
         `,
 );
 
+// Labs paid content only
+const labsStyle = css`
+    p,
+    li {
+        font-weight: 700;
```

Comment:
I'm not sure why these fonts need to be bold, maybe we can remove this styling to keep the consistency @zeftilldeath? 

Otherwise, for a future PR, this is possibly something we need to add to `typography.ts`, a boolean for whether we want to use the bold version @nicl ?

Reply:
This styling is based on frontend web: 

![image](https://user-images.githubusercontent.com/32312712/57533034-af6a8c80-7334-11e9-9b9b-88e39f823abd.png)

Reply:
Yup, don't feel it's consistent there either 😄